src/components/common/PostalCode/index.tsx

```diff
@@ -1 +1,2 @@
-export { default } from './PostalCodeInput'
+export { default } from './PostalCodeBar'
+export { default as PostalCodeInput } from './PostalCodeInput'
```

Comment:
as `PostalCode` component is part of the Regionalization feature, it should be inside `src/components/regionalization` folder, isn't it?